code/game/gamemodes/objective.dm

```diff
@@ -588,6 +608,10 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 	return steal_target
 
 /datum/objective/steal/update_explanation_text()
+	if(delayed_objective)
+		explanation_text = "Your objective is to steal a high-security item. Please hold for further details."
+		return
```

Comment:
"High-security" doesnt feel like it fits, "High-value" is the term in Space Law, so it should probably be used here too.
```suggestion
		explanation_text = "Your objective is to steal a high-value item. Please hold for further details."
```

Reply:
I somehow goofed the line in the review, guh. But point still stands.

Reply:
Done!